packages/contentstack-migration/src/commands/cm/stacks/migration.js

```diff
@@ -129,6 +129,10 @@ class MigrationCommand extends Command {
     set(MANAGEMENT_SDK, mapInstance, stackSDKInstance);
     set(MANAGEMENT_CLIENT, mapInstance, APIClient);
 
+    if (!(await installModules(filePath, multi))) {
+      this.log(`Cannot proceed with migration as there will be module not found issue as package.json is not created`);
```

Comment:
"Failed to install the dependencies of the given scripts."  please change the text , then we can exit

Reply:
added